tidy3d/components/source.py

```diff
@@ -9,10 +9,20 @@
 
 from .base import Tidy3dBaseModel, cached_property
 
-from .types import Direction, Polarization, Ax, FreqBound, ArrayFloat1D, Axis, PlotVal
+from .types import (
+    Direction,
+    Polarization,
+    Ax,
```

Comment:
please split this import into two lines
```py
from .types import x, y, z
from .types import a, b, c
```

just an arbitrary code style thing, for consistency.